3441.xml

```diff
@@ -0,0 +1,243 @@
+                    multiple value, the number of resource instances is
+                    also random.</Description>
+            </Item>
+            <Item ID="2">
+                <Name>Clear values</Name>
+                <Operations>E</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type></Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>
+<![CDATA[Clear all values : 
+ - all multiple resource as empty resource
+ - all number to 0
+ - String to empty string
+ - boolean to false,
+ - opaque to empty byte array,
+ - time to an 1st, 1970 in the UTC time zone
+ - objlink to null link]]>
+                </Description>
+            </Item>
+
+            <Item ID="10">
+                <Name>String Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>String</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "initial value".
+                </Description>
+            </Item>
+            <Item ID="20">
+                <Name>Integer Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description> Initial value must be "1024".
+                </Description>
+            </Item>
+            <Item ID="30">
+                <Name>Float Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Float</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "3.14159".
+                </Description>
+            </Item>
+            <Item ID="40">
+                <Name>Boolean Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Boolean</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "true".</Description>
+            </Item>
+            <Item ID="50">
+                <Name>Opaque Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Opaque</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "0x0123456789ABCDEF".
```

Comment:
How is the value encoded? In binary I suppose (not as ASCII characters)?

Reply:
I'm not sure to get the point.
This is an Opaque resource, so not limited to ASCII chars.
About encoding it depends of ContentFormat used by the request.

(I feel I missed something :thinking:)

Reply:
I meant how the value `"0x0123456789ABCDEF"` is encoded to the resource. I assume you mean this is a sequence of 16 octets that have values (shown in hex) "0x0123456789ABCDEF". Alternative interpretation would be sequence of ASCII characters.

Reply:
Correct, 0x0123456789ABCDEF is the byte array value in Hexadecimal. 
Do you feel the description is not clear enough ? 

Should I replace by : 

> Initial value must be 0123456789ABCDEF (Hexadecimal notation)

OR 

> Initial value must be 0x0123456789ABCDEF (Hexadecimal notation)

OR

any other better idea ?

Reply:
Both are better than the original but if you want to be even more clear I would suggest something like: 
> Sequence of 16 octets with value (shown in hex) "0x0123456789ABCDEF".

I wonder if there is some precedent of such description in another object we could replicate here. But you probably know best what is the most clear description for developers.

Reply:
I changed it (commit daabe13)